### PR TITLE
MWPW-123950 Set up repository to mirror bacom

### PR DESCRIPTION
**Pull request by JasonHowellSlavin**
* Adding files, folders, configs
* Adjusted some configs and the readme to reflect bacom-blog

Resolves: [MWPW-123950(https://jira.corp.adobe.com/browse/MWPW-123950)

**Test URLs:**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+* Add your specific features or fixes
+
+Resolves: [MWPW-NUMBER](https://jira.corp.adobe.com/browse/MWPW-NUMBER)
+
+**Test URLs:**
+- Before: https://main--bacom--adobecom.hlx.page/?martech=off
+- After: https://<branch>--bacom--adobecom.hlx.page/?martech=off
```
Comment (meganthecoder) [inline .github/pull_request_template.md]: URL should be updated. Is it bacom-blog?

File: scripts/scripts.js [hunk 2]
```diff
@@ -12,24 +12,114 @@
...
+  },
+  live: {
+    pdfViewerClientId: '23bd4fff42fc4b4da38b3d89492a0abc',
+    pdfViewerReportSuite: 'adbadobedxqa',
+  },
+  prod: {
+    edgeConfigId: '65acfd54-d9fe-405c-ba04-8342d6782ab0',
+    pdfViewerClientId: '4520c0edfbf147158758d71d18765fec',
+    pdfViewerReportSuite: 'adbadobenonacdcprod,adbadobedxprod,adbadobeprototype',
+  },
```
Comment (meganthecoder) [inline scripts/scripts.js]: Should check on these configs-maybe make a ticket for it? The pdf client ids are specific to the bacom urls so we'll need to generate new ones for this site. Not sure if the report suites are the same or not. We'd need to ask Malik. 

Not sure about the other configs

File: scripts/scripts.js [hunk 3]
```diff
@@ -12,24 +12,114 @@
...
+    cn: { ietf: 'zh-CN', tk: 'puu3xkp' },
+    hk_zh: { ietf: 'zh-HK', tk: 'jay0ecd' },
+    tw: { ietf: 'zh-TW', tk: 'jay0ecd' },
+    jp: { ietf: 'ja-JP', tk: 'dvg6awq' },
+    kr: { ietf: 'ko-KR', tk: 'qjs5sfm' },
+    // Langstore Support.
+    langstore: { ietf: 'en-US', tk: 'hah7vzn.css' },
   },
+  // geoRouting: 'on',
+  productionDomain: 'business.adobe.com',
```
Comment (meganthecoder) [inline scripts/scripts.js:106]: Is this the correct domain?

File: scripts/scripts.js (hunk 2 above)
Comment (JasonHowellSlavin) [reply scripts/scripts.js]: Checked with Dave, he said everything should be the same, there is a comment on the ticket. I'll remove PDF viewer for now, and we can add it if needed.

File: scripts/scripts.js (hunk 3 above)
Comment (JasonHowellSlavin) [reply scripts/scripts.js:106]: It is what the sharepoint is pointing to at the moment, but I will ask Malik and Dave. Preview of the documents on sharepoint is not working atm.

**Review by Brandon32 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
-2. Click "[Use this template](https://github.com/adobecom/milo-college/generate)" Github button on this project.
+## Developing
+1. Install the [Helix CLI](https://github.com/adobe/helix-cli): `sudo npm install -g @adobe/helix-cli`
+2. Run `hlx up` this repo's folder. (opens your browser at `http://localhost:3000`)
+3. Open this repo's folder in your favorite editor and start coding.
 
-From your newly created project
+### Husky Pre-Commit
+1. After pulling the main branch, make sure to run `npm install` to get the husky pre-commit package.
+2. Husky will run the linter before your commit, and will fail if the linter fails.
```
Comment (Brandon32) [inline README.md:11]: Need to update that Husky is running tests.

**Comment by JasonHowellSlavin** [comment]
This setup was 404ing with local hlx up, closing, though the next branch is nearly identical.

**Pull request closed without merging**